src/components/KCopy/KCopy.vue

```diff
@@ -0,0 +1,263 @@
+<template>
+  <div class="k-copy">
+    <span
+      v-if="badge"
+      class="copy-badge-text"
+    >
+      {{ badgeText }}
+    </span>
+    <div
+      :class="[
+        'copy-container',
+        truncateStyles,
+        badgeStyles
+      ]"
+    >
+      <KTooltip
+        v-if="format !== 'hidden'"
+        class="copy-text"
```

Comment:
When I hover over this element, I always have a pointer cursor even if there is no tooltip

Reply:
Fixed.